uitools/widgets/Esri/ArcGISRuntime/Toolkit/BasemapGallery.cpp

```diff
@@ -0,0 +1,54 @@
+#include "BasemapGallery.h"
+#include "ui_BasemapGallery.h"
+
+
+//Toolkit controllers
+
+namespace Esri {
+namespace ArcGISRuntime {
+namespace Toolkit {
+BasemapGallery::BasemapGallery(GeoModel* geomodel, QWidget *parent) :
+  QFrame(parent),
+  m_ui(new Ui::BasemapGallery), m_controller(new BasemapGalleryController(this))
```

Comment:
:x:  This constructor effectively prevents the `BasemapGallery` from being used in designer. As Qt designer expects constructors to be of the form `BasemapGallery(QWidget* parent = nullptr)`.

You'll have to either remove teh `Geomodel* geomodel` parameter from this constructor, or make an additional constructor. 

We also need a getter/setter for the `Geomodel`.